yash.md

```diff
@@ -0,0 +1,82 @@
+# Yet Another SDR Handbook
+
+_A combination index, glossary, and high level guided tour for the Stanford Digital Repository_
+
+* Access Rights for digital content
+  * What's an APO?
+  * Where are access restrictions computed when interacting with a specific object? - various codebases use the logic provided by the dor-rights-auth gem:  https://github.com/sul-dlss/dor-rights-auth
```

Comment:
I would format this so the answers start on a separate line and/or have different formatting.  It will make it much more skimmable.